Dockerfile.deps

```diff
@@ -47,18 +69,17 @@ RUN curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain 1.56.1
 RUN source /root/.cargo/env \
  && mkdir -p /srv/explorer/electrs{,_liquid} \
  && git clone --no-checkout https://github.com/blockstream/electrs.git \
- && cd electrs \
+ && (cd electrs \
  && git checkout 703c6a20d52b61a234a18812503bd754d448992a \
  && cp contrib/popular-scripts.txt /srv/explorer \
  && cargo install --root /srv/explorer/electrs_bitcoin --locked --path . --features electrum-discovery \
  && cargo install --root /srv/explorer/electrs_liquid --locked --path . --features electrum-discovery,liquid \
- && cd .. \
```

Comment:
The `rm` for the `electrs` directory on the line below wouldn't work well without the `cd ..`.

One way to fix it is to close the `)` the second `cargo install` instead of after `strip`.